client/task-list/tasks/payments/index.js

```diff
@@ -284,19 +284,26 @@ class Payments extends Component {
 							</CardBody>
 							<CardFooter isBorderless>
 								{ container && ! isConfigured ? (
-									<Button
-										isPrimary={ key === recommendedMethod }
-										isSecondary={
-											key !== recommendedMethod
-										}
-										isBusy={ busyMethod === key }
-										disabled={ busyMethod }
-										onClick={ () =>
-											this.handleClick( method )
-										}
-									>
-										{ __( 'Set up', 'woocommerce-admin' ) }
-									</Button>
+									<div>
```

Comment:
the wrapper div is needed to avoid a bug with card footer css that removes padding on last child.